components/script/script_runtime.rs

```diff
@@ -122,6 +122,7 @@ impl fmt::Debug for CommonScriptMsg {
 }
 
 /// A cloneable interface for communicating with an event loop.
+#[allow(clippy::result_unit_err)]
```

Comment:
Can we move this to be above the `fn send()` method so that we don't inadvertently `allow` new methods in this trait?